src/Entity/User.php

```diff
@@ -75,6 +75,13 @@ class User extends FOSUser implements TwoFactorInterface, TrustedDeviceInterface
      */
     protected $trash;
 
+    /**
+     * @var array|null
+     *
+     * @ORM\Column(type="json", options={"jsonb": true}, nullable=true)
+     */
+    protected $keys;
```

Comment:
I think we should have two different fields on DB, it will give more consistency

Reply:
Maybe layter? For keys quantity and its names too fuzzily.

Reply:
Maybe later you can not change this logic without pain. Right now you have two keys - encryptedPrivateKey, publicKey